investigator/base/deploymentconfig.yaml

```diff
@@ -59,6 +59,40 @@ spec:
                 secretKeyRef:
                   name: postgresql
                   key: database-name
+            - name: THOTH_MIDDLETIER_NAMESPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: thoth
+                  key: middletier-namespace
+            - name: THOTH_INFRA_NAMESPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: thoth
+                  key: infra-namespace
+            - name: THOTH_CEPH_BUCKET
+              valueFrom:
+                configMapKeyRef:
+                  key: bucket-name
+                  name: ceph
+            - name: THOTH_CEPH_BUCKET_PREFIX
+              valueFrom:
+                configMapKeyRef:
+                  key: bucket-prefix
+                  name: ceph
+            - name: THOTH_S3_ENDPOINT_URL
+              valueFrom:
+                configMapKeyRef:
+                  key: host
+                  name: ceph
+            - name: THOTH_DEPLOYMENT_NAME
+              valueFrom:
+                configMapKeyRef:
+                  key: deployment-name
+                  name: thoth
+            - name: KUBERNETES_API_URL
+              value: "https://kubernetes.default.svc.cluster.local"
+            - name: KUBERNETES_VERIFY_TLS
+              value: "0"
```

Comment:
Is this in cluster issue?

Reply:
> Is this in cluster issue?

I saw this error message: 

```
/template.openshift.io/v1/namespaces/thoth-test-core/processedtemplates (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:897)'),))`
```

Reply:
Sounds like certification issue in the cluster.

Reply:
@fridex , any suggestion on how to resolve this in cluster ?
or should we go ahead with this solution of disabling it?

Reply:
Let's disable it for now.

Reply:
thanks for looking into this @fridex